facilitator/src/aggregation.rs

```diff
@@ -125,6 +127,7 @@ impl<'a> BatchAggregator<'a> {
                 aggregation_start_time: self.aggregation_start.timestamp_millis(),
                 aggregation_end_time: self.aggregation_end.timestamp_millis(),
                 packet_file_digest: invalid_packets_digest.as_ref().to_vec(),
+                total_individual_clients,
```

Comment:
Could you please update the test in `tests/integration_tests.rs` to check that this value is handled correctly?